Helper/ArrayReporter.php

```diff
@@ -65,6 +66,7 @@ public function onAfterRun(CheckInterface $check, ResultInterface $result, $chec
             'status' => $status,
             'status_name' => $statusName,
             'service_id' => $checkAlias,
+            'duration' => round(microtime(true) - $this->checkRunTime, 2),
```

Comment:
```suggestion
            'duration' => microtime(true) - $this->checkRunTime,
```

I think we shouldn't make assumptions on the accuracy here.

Reply:
I agree with you. fixed.